### PR TITLE
PR for issue 434, making rasa action server listen on a specific inte…

### PR DESCRIPTION
**Pull request by bastbnl**
…rface

**Proposed changes**:
- Restrict the network interfaces used by rasa to one specific address instead of 0.0.0.0
- Fixes the network address logged at server startup

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [ ] added some tests for the functionality
- [x] updated the documentation
- [x] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa-sdk/tree/main/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa-sdk#code-style) for instructions)

### CODE REVIEW

**Comment by sara-tagger** [comment]
Thanks for submitting a pull request 🚀 @wochinge will take a look at it as soon as possible ✨

File: changelog/434.improvement.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+- The action server can now listen on a specific network address using the environment variable `SANIC_HOST` instead
+- Corrected url which is logged at action server startup
```
Comment (wochinge) [inline changelog/434.improvement.md:2]: Is there meant to be a fullstop between `instead` and `Corrected?

Comment (bastbnl) [reply changelog/434.improvement.md]: Thanks for reviewing @wochinge. The answer is no and I corrected the text

**Review by wochinge — APPROVED** [review]
💯

**Pull request merged**